### DISCUSSION TITLE
Thanks! for iOS 3.4.3

### DISCUSSION THREAD

**Discussion by KarmaBeast** [nrubin29/finale#33] [General]
The new version of finale for lastfm looks really good, and the fact that the theme applies to everything. No longer the high contrast black background! I just updated and I'm so glad the black amoled background is gone now 🫡

The app looks way prettier now! Thanks a lot for maintaining and updating it! ❤️ @nrubin29
![image](https://github.com/nrubin29/finale/assets/73917917/95e2cb63-e9f2-4c0c-87bd-331fcd740039)
![image](https://github.com/nrubin29/finale/assets/73917917/bdfcce66-1d3e-4677-8d82-03aabe9838dd)

**Comment by nrubin29** [comment]
Thanks! The credit for this goes to the Flutter team :)

**Comment by KarmaBeast** [comment]
Could you please look into this error im getting sometimes when opening the app? (It forces me to pull down and refresh in order to work)
![image](https://github.com/user-attachments/assets/0b39acea-1f52-4c8d-ab1f-2a520ea98adb)
